README.md

```diff
@@ -236,6 +236,15 @@ Default: `true`
 
 Option to prevent the livereload if the executed tasks encountered an error.  If set to `false`, the livereload will only be triggered if all tasks completed successfully.
 
+#### options.silent
+Type: `Boolean`
+Default: `false`
+Aliases: `silently`
```

Comment:
Came to same conclusion. See second commit. I provided an alias though for better consistency, in case `silently` is used elsewhere (e.g. when using grunt-contrib-connect). If you consider this feature creep lets remove the alias though.